src/Core/Silk.NET.BuildTools/Cpp/ComVtblProcessor.cs

```diff
@@ -188,23 +188,14 @@ public static void Implement(StringBuilder sb, Function function, Struct parent,
                 }
             }
 
-            if (function.ReturnType.ToString() != "void")
-            {
-                sb.Append(ind + "ret = ");
-            }
-            else
-            {
-                sb.Append(ind);
-            }
-
             var conv = function.Convention switch
             {
-                CallingConvention.Winapi => throw new NotImplementedException(),
+                CallingConvention.Winapi => string.Empty,
```

Comment:
Remove option from switch? Default is already string.Empty
```suggestion
-                CallingConvention.Winapi => string.Empty,
```

Reply:
I can do this if we really, but Rider/ReSharper complains about it!

Reply:
Nah just a suggestion, the compiler probably optimizes it anyway